integration-cli/docker_api_containers_test.go

```diff
@@ -450,6 +451,11 @@ func (s *DockerSuite) TestContainerAPITop(c *check.C) {
 		c.Fatalf("expected `USER` at `Titles[0]` and `COMMAND` at Titles[10]: %v", top.Titles)
 	}
 	c.Assert(top.Processes, checker.HasLen, 2, check.Commentf("expected 2 processes, found %d: %v", len(top.Processes), top.Processes))
+
+	// sort by command name to make sure order stays the same in case of PID rollover
+	sort.Slice(top, func(i, j int) bool {
+		return top.Processes[i][10] < top.Processes[j][10]
+	}
```

Comment:
Missing a closing `)` here? 

```
00:36:47 # github.com/docker/docker/integration-cli
00:36:47 integration-cli/docker_api_containers_test.go:458:3: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:459:66: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:460:55: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:461:1: expected operand, found '}'
00:36:47 integration-cli/docker_api_containers_test.go:471:2: missing ',' in argument list
00:36:47 integration-cli/docker_api_containers_test.go:473:11: missing ',' in argument list
00:36:47 integration-cli/docker_api_containers_test.go:474:30: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:475:121: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:477:2: expected operand, found 'if'
00:36:47 integration-cli/docker_api_containers_test.go:478:100: missing ',' before newline in argument list
00:36:47 integration-cli/docker_api_containers_test.go:479:2: expected operand, found '}'
```

Reply:
whoops, I have committed the wrong version. fixed

Reply:
LOL, already was thinking: "this looked different"?